_posts/uportal/2017-08-30-summer-2017-uportal-roadmap-update.md

```diff
@@ -0,0 +1,33 @@
+---
+layout:     post
+title:      Summer 2017 uPortal Roadmap Update
+summary:    Primarily summarizing the Open Apereo 2017b uPortal Roadmap BOF.
+tags:       [uPortal]
+---
+
+# Summary
+Almost two dozen conference attendees came to the uPortal Roadmap BOF at the Open Apereo 2017 conference. Prior to the conference, members of the community were invited to submit and comment on roadmap ideas in a shared google doc (https://goo.gl/wZ7VTC). 15 items were suggested but during the BOF we focused on which ones we anticipated someone would actually devote time and resources to. In the end, we anticipate making progress over the next year on the 8 items listed below.
+
+# Release of uPortal 5
+This long awaited release includes a refactoring of the build system using a Gradle-based build in the uPortal-start repository. We are currently aiming for a mid-September release candidate and feature freeze followed by a GA release by the end of October.
+
+# Semantic versioning and releasing often
+While no specific action items were identified, we renewed our commitment to ship releases more often and use semantic versioning when doing so. We should not be afraid of doing more frequent major releases.
+
+# Better web service API documentation
+Most of the existing web service APIs are not well documented. This makes them harder to use or sometimes even be aware of. There was a commitment to documenting new APIs that are developed. UW-Madison committed to documenting APIs it depends on.
+
+# Easier adoption: Complete and accurate documentation
+A work in progress, we are looking at shifting uPortal 5 documentation to github.io with an initial focus on installing and configuring core uPortal. This remains in early stages and the details are in flux. Discussion continues on the uportal email lists.
```

Comment:
Consider hyperlinking the in-flux early stage uPortal documentation on `github.io`. Consider hyperlinking the discussion on email lists.